src/main/java/io/antmedia/streamsource/StreamFetcherManager.java

```diff
@@ -89,7 +90,12 @@ public void setRestartStreamFetcherPeriod(int restartStreamFetcherPeriod) {
 
 
 	public StreamFetcher startStreaming(Broadcast broadcast) {	
-
+		if(scope.getContext().hasBean(IResourceMonitor.BEAN_NAME)) {
```

Comment:
I think in all cases resource monitor will exist in the streams so don't check with hasBean and fail if it does not exist